src/main/java/nl/knaw/dans/migration/core/tables/ActualDataset.java

```diff
@@ -28,7 +28,7 @@
 @IdClass(ActualDatasetKey.class)
 @Table(name = "actual_datasets",
        indexes = {
-           @Index(name = "doi_index", columnList = "doi")
+           @Index(name = "ad_doi_index", columnList = "doi")
```

Comment:
Note that the old index without the prefix is not dropped from an existing table.